### PR TITLE
Fix responsiveness in iframe shortcode

### PR DESCRIPTION
**Pull request by davisshaver**
https://twitter.com/evilgaywitch/status/623905861946093568

### CODE REVIEW

File: inc/shortcodes/class-iframe.php [hunk 1]
```diff
@@ -50,9 +50,11 @@ public static function callback( $attrs, $content = '' ) {
 		}
 
 		return sprintf(
-			'<iframe src="%s" width="%s" height="%s" frameborder="0" scrolling="no" class="shortcake-bakery-responsive"></iframe>',
+			'<iframe src="%s" width="%s" height="%s" data-true-width="%s" data-true-height="%s" frameborder="0" scrolling="no" class="shortcake-bakery-responsive"></iframe>',
```
Comment (danielbachhuber) [inline inc/shortcodes/class-iframe.php]: Oh, this is how the responsiveness JS knows which proportion to maintain. I guess we were missing it before?

**Comment by davisshaver** [comment]
Previous height/width defaults were:
```

**Comment by davisshaver** [comment]
@danielbachhuber can I get a sanity check on this?

**Comment by danielbachhuber** [comment]
> can I get a sanity check on this?

Seems pretty reasonable.

**Comment by davisshaver** [comment]
@goldenapples would you mind giving me a review/merge?

**Comment by danielbachhuber** [comment]
:monkey:

**Pull request merged**